docs/articles/documentation/test-api/assertions/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: docs
+title: Assertions
+permalink: /documentation/test-api/assertions/
+checked: false
+---
+# Assertions
+
+To check if the state of the tested webpage matches the one you expect to see, use *assertions*.
+
+TestCafe provides a comprehensive [set of assertions](assertion-api.md) and all of them based on Behavior Driven Development style (BDD-style).
+
+This topic contains the following sections.
+
+* [Assertion timeout](#assertion-timeout)
+* [Assertion options](#assertion-options)
+  * [options.timeout](#optionstimeout)
+
+The following example demonstrates how to check whether two values are equal or not.
+
+```text
+    await t.expect(actual).eql(expected, message, options);
+```
+
+Parameter              | Type     | Description
+---------------------- | -------- | -------------------------------------------------------------------------------
+`actual`                 | Any type | The comparison value
+`expected`                 | Any type | Expected value
+`message`&#160;*(optional)* | String   | Assertion message, will be indicated in the report in case of error.
+`options`&#160;*(optional)* | Object   | See [Options](#assertion-options).
+
+Assertions allow you to check values of the standard types (String, Number, Object and etc.). You can also use assertions
+to check states of the webpage's elements.
+o check state of any element on the page in assertion, you can use properties and methods of
+[DOM Node Snapshot](../selecting-page-elements/selectors.md#dom-node-snapshots)
+or use [Snapshot API Shorthands](../selecting-page-elements/selectors.md#snapshot-api-shorthands).
+
+The sample below demonstrates how to use assertions to check element states.
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `Example page`
+    .page `http://devexpress.github.io/testcafe/example/`;
+
+test('Check property of element', async t => {
+    const developerNameInput = Selector('#developer-name');
+
+    await t
+        .expect(developerNameInput().value).eql('', 'input is empty') // Assertion with DOM node snapshot property
```

Comment:
'Assertion with snapshot API shorthands' is below. Do you offer completely get rid of example with DOM node snapshot property?